README.md

```diff
@@ -0,0 +1,41 @@
+# Emotion
+Quickly determine the emotion on a face. Trained on tens of thousands of images.
+
+**Possible Use Cases**
+  * Adding emotion classification to a face detection system
+  * Targeted messaging to your users based on their emotional responses
+  * Measuring the emotional response of an audience during a movie
```

Comment:
Change last one to:

- Automated filter recommendations on image messaging apps like Instagram
- Smart tagging in photo gallery apps.